hummingbot/connector/exchange/bittrex/bittrex_api_user_stream_data_source.py

```diff
@@ -127,18 +126,19 @@ async def listen_for_user_stream(self, ev_loop: asyncio.BaseEventLoop, output: a
 
                 async for raw_message in self._socket_user_stream(self._websocket_connection):
                     decode: Dict[str, Any] = self._transform_raw_message(raw_message)
+                    self.logger().debug(f"Got ws message {decode}")
```

Comment:
This seems as a log introduced only for the issue investigation

Reply:
This is true, but I left it here because I think these messages can provide valuable information for any debugging, so when QA tests, enabling this can be useful